libbpf-rs/src/program.rs

```diff
@@ -996,6 +997,23 @@ impl<'obj> ProgramMut<'obj> {
         Ok(link)
     }
 
+    /// Attach this program to [netfilter programs](https://lwn.net/Articles/925082/)
+    pub fn attach_netfilter(&self, netfilter_opt: netfilter::NetfilterOpts) -> Result<Link> {
```

Comment:
I'd think that for the sake of consistency, it would be better to name this method `attach_netfilter_with_opts`. What do you think?